SPECS/kernel/config

```diff
@@ -7405,6 +7408,7 @@ CONFIG_CRYPTO_ALGAPI=y
 CONFIG_CRYPTO_ALGAPI2=y
 CONFIG_CRYPTO_AEAD=y
 CONFIG_CRYPTO_AEAD2=y
+CONFIG_CRYPTO_SIG=y
```

Comment:
result of: https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/21c7e4587e29c026352c29a226a90e2ce1ce72b2